lib/_emerge/EbuildMetadataPhase.py

```diff
@@ -139,6 +159,21 @@ def _start(self):
 
         self._proc = retval
 
+    def _async_start_done(self, future):
+        future.cancelled() or future.result()
+        if self._was_cancelled():
+            pass
```

Comment:
Am I missing why we write it like this?

Why not just:
`if not self._was_cancelled() and future.cancelled():`

Reply:
Your version is equivalent so I updated it to use that.